Exercise_D_E/styles.css

```diff
@@ -0,0 +1,86 @@
+* {
+    margin: 0;
+}
+
+nav {
+    background-color: white;
+}
+
+.first-container {
+    position: relative; /*positioned relative to its normal position.*/
+    text-align: center;
+    color: white;
+}
+
+.first-image {
+    filter: brightness(50%); /* filter: brightness property to darken the image by 50% */
+    width: 100%; /*width property is set to a percentage and the height property is set to "auto", the image will be responsive and scale up and down*/
+    height: auto;
+
+}
+
+.centered-text {
+    position: absolute; /*positioned relative to the nearest positioned ancestor which is the container div*/
+    top: 50%;
+    left: 53%;
+    transform: translate(-50%, -50%);
+    font-size: 40px ;
+
+    }
+
+
+.button {
+    position: absolute;
+    color: white;
+    font-size: 20px;
+    top: 60%;
+    left: 50%;
+    background-color: #7ed320;
+    padding: 5px 15px;
+    border-radius: 25px;
+}
+
+.second-container {
+    height: 200px;
+    background-color:#cee7b4;
+    display: flex;
+    justify-content: center; 
+    align-items: center;
+}
+
+.items {
+    height: 100px;
+    width: 150px; 
+    margin-left: 80px;
+    font-size: 25px;
+    display: flex;
+    align-items: center;
+
+}
+
+@media screen and (max-width : 450px){ /* if screen size is less than 450px than following media query will hva affect*/
+    .nav-image{
+        width: 100px;
+        height: 30px;
+    }
+    
+    .centered-text{
+        font-size: 22px;
+    }
+
+    .button{
+        font-size: 10px;
+    }
+
+    .items{
+        margin-left: 15px;
+        font-size: 15px;
+
+    }
+    .small-icon{
+        height: 75px;
+        width: 75px;
+    }
+
+    
+}
```

Comment:
Hello Asis
I would suggest when working with Media Queries, you can make use of em and rem units to make it more responsive but as its 450px this looks good on iPhone's and a Samsung Phone though the cantering of the text and the button could be further adjusted.